gyp_xwalk

```diff
@@ -351,6 +351,9 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if sys.platform == 'win32':
```

Comment:
What I said in a previous review is that you should be able to do this in `build/common.gypi` to override the defaults from `src/third_party/widevine/cdm/widevine_cdm.gyp`:

```python
'conditions': [
  # Enable Widevine support by default on Windows.
  ['OS=="win"', {
    'enable_widevine%: 1',
    }, {
    'enable_widevine%: 0',
  }],
],
```